src/sentry/tagstore/snuba/backend.py

```diff
@@ -217,25 +239,25 @@ def get_top_group_tag_values(self, project_id, group_id, environment_id, key, li
         }
         conditions = [[tag, '!=', '']]
         aggregations = [
-            ['count()', '', 'count'],
+            ['count()', '', 'times_seen'],
             ['min', SEEN_COLUMN, 'first_seen'],
             ['max', SEEN_COLUMN, 'last_seen'],
         ]
 
         result = snuba.query(start, end, [tag], conditions, filters,
-                             aggregations, limit=limit, orderby='-count')
-
-        return [ObjectWrapper({
-            'times_seen': val['count'],
-            'first_seen': val['first_seen'],
-            'last_seen': val['last_seen'],
-            'key': key,
-            'value': name,
-            'group_id': group_id,
-        }) for name, val in six.iteritems(result)]
+                             aggregations, limit=limit, orderby='-times_seen')
+        return [
+            GroupTagValue(
+                group_id=group_id,
+                key=key,
+                value=value,
+                **fix_tag_value_data(data)
+            ) for value, data in six.iteritems(result)
+        ]
 
+    # TODO: Coming back to this later
+    """
```

Comment:
Not sure what the longer term plan for dealing with this is until we figure out the top-K count thing.